components/teams-service/storage/postgres/postgres.go

```diff
@@ -439,9 +402,9 @@ func (p *postgres) GetTeamsForUser(ctx context.Context, userID string) ([]storag
 	rows, err := p.db.QueryContext(ctx,
 		`SELECT t.id, t.name, t.description, t.projects, t.created_at, t.updated_at
 		FROM teams t
-		LEFT JOIN teams_users_associations tu ON tu.team_id=t.id
+		LEFT JOIN teams_users_associations tu ON tu.team_db_id=t.db_id
 		WHERE tu.user_id=$1 AND projects_match(t.projects, $2::TEXT[])
-		GROUP BY t.id`,
+		GROUP BY t.id, t.name, t.description, t.projects, t.created_at, t.updated_at`,
```

Comment:
💭 Why the added groupings?

Reply:
I was getting an error that name was required without it when I ran the db tests.

Reply:
If i delete the entire group by we're fine so 💥